Facebook.Unity/Assets/Facebook/Scripts/Mobile/UWP/WindowsFacebook.cs

```diff
@@ -0,0 +1,331 @@
+﻿using UnityEngine;
+using Facebook.Unity;
+using Facebook.Unity.Mobile;
+using System;
+using System.Collections.Generic;
+#if UNITY_UWP
```

Comment:
I don't see this mentioned in the unity docs 
http://docs.unity3d.com/Manual/PlatformDependentCompilation.html

Should this be UNITY_WSA?

Reply:
@swiese 
https://docs.unity3d.com/2017.4/Documentation/Manual/PlatformDependentCompilation.html
2017 version defines it